src/pages/workspaces/workspace/analysis/utils/cost-utils.ts

```diff
@@ -130,16 +140,16 @@ export const getGalaxyCost = (app, dataDisk) => {
  *   be complicated to calculate that shared cost dynamically. Therefore, we are being
  *   conservative by adding default nodepool cost to all apps on a cluster.
  */
-export const getGalaxyComputeCost = app => {
+export const getGalaxyComputeCost = (app:App) => {
   const appStatus = app?.status?.toUpperCase()
```

Comment:
We should remove the .toUpperCase here. It converts the appStatus to a string.
That way in the below switch(appStatus) it will have the correct type protection.